src/consensus/aft/async_execution_coordinator.h

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please rename this file to `async_executor.h`, since that's now the main type it defines.